eservice/lib/libpdo_enclave/contract_request.cpp

```diff
@@ -145,9 +147,9 @@ ContractResponse ContractRequest::process_initialization_request(void)
     // the only reason for the try/catch here is to provide some logging for the error
     try
     {
-        GipsyInterpreter interpreter;
+        GipsyInterpreter *interpreter = worker_->interpreter_;
 
-        // interpreter.create_initial_contract_state(contractid, creatorid, contractcode, message,
+        // interpreter->create_initial_contract_state(contractid, creatorid, contractcode, message,
```

Comment:
Can you just delete this comment instead of editing it? It should go away anyway.